package relacion11ConMapa.exception;

public class AlumnoContenidoException extends Exception {
	public AlumnoContenidoException() {
		super("El alumno ya está contenido en el equipo.");
	}

}
